### PR TITLE
PS-8901 Remove encryption thread topics from 8.1 docs (8.1)

### PR DESCRIPTION
**Pull request by patrickbirch**
Edited and rewrote encryption topics
Fixed snippets for several topics

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/encrypt-binary-relay-log-files.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+To review if a binary log file is encrypted or not, use the `SHOW BINARY LOGS` statement.
+
+If the server is running, the `BINLOG_ENCRYPTION_ADMIN` privilege is required to enable or disable encryption.
+
+## `binlog_encryption`
+
+| Option       | Description               |
+|--------------|---------------------------|
+| Command-line | `--binlog-encrytion[= {ON \| OFF}]` |
```
Comment (inikep) [inline docs/encrypt-binary-relay-log-files.md]: `encrytion` should be changed to `encryption`

File: docs/verify-encryption.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ INFORMATION_SCHEMA.SCHEMATA table:
 
 ```{.bash data-prompt="mysql>"}
 mysql> SELECT SCHEMA_NAME, DEFAULT_ENCRYPTION FROM
-INFORMATION_SCHEMA.SCHEMATA WHERE DEFAULT_ENCRYPTION='YES';
+INFORMATION_SCHEMA.SCHEMATA WHERE DEFAULT_ENCRYPTION='Y';
```
Comment (inikep) [inline docs/verify-encryption.md]: Should stay `DEFAULT_ENCRYPTION='YES'`.

**Review by inikep — CHANGES_REQUESTED** [review]

File: docs/encrypt-binary-relay-log-files.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Encrypt binary log files and relay log files
```
Comment (percona-ysorokin) [inline docs/encrypt-binary-relay-log-files.md:1]: This is an upstream variable. Why do we have a separate section for it in Percona Docs?

File: docs/encrypt-logs.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Log encryption
+
+Describes the redo log encryption and the undo log encryption.
+
+## Redo Log encryption
+
+Use the `innodb_redo_log_encrypt` option to enable or disable redo log data encryption. By default, the encryption of the redo log is disabled.
```
Comment (percona-ysorokin) [inline docs/encrypt-logs.md:7]: Likewise, how are our `innodb_redo_log_encrypt ` and `innodb_undo_log_encrypt` different from the upstream?

File: docs/encrypt-binary-relay-log-files.md (hunk 3 above)
Comment (patrickbirch) [reply docs/encrypt-binary-relay-log-files.md:1]: One of our documentation initiatives is that we have complete docs.

File: docs/encrypt-logs.md (hunk 4 above)
Comment (patrickbirch) [reply docs/encrypt-logs.md:7]: One of our documentation initiatives is that we have complete docs.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**